File: desktop-staff/reservation-winforms/DTO/table/TableUpdate.cs
using Newtonsoft.Json;
using System;

namespace reservation_winforms.DTO.table
{
    public class TableUpdate
    {
        [JsonProperty("tableId")]
        public long TableId { get; set; }

        [JsonProperty("status")]
        public string Status { get; set; }
    }
}

File: backend-api/reservation-api/src/main/java/com/hcmute/reservation/model/dto/table/TableUpdate.java
package com.hcmute.reservation.model.dto.table;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class TableUpdate {
    private Long tableId;
    private String status; // "AVAILABLE", "OCCUPIED", "OVERSTAY"
}
